README.md

```diff
@@ -1 +1,72 @@
-# Capstone-project-1
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# restuarant-menu
+
+> This website is about how to order food online and also satisfy your cravings. we deliver at your door step.
+
+
+## Built With
+
+- HTML/CSS3 
+- Javascript
+- Linters (Webhint, Eslint, Stylelint)
+- Visual Studio
+
+## Live Demo (if available)
+
+[Live Demo Link](https://rmjspecial.github.io/Capstone-project-1/)
+[Video Link](https://www.loom.com/share/9ad514e15fb44746a69df0271a99e3ea)
+
+
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Web Browser
+- Command Line (e.g. Git Bash)
+- Code Editor
+
+### Setup
+- Make a clone of the repository or download as zip by click on code button
+
+### Install
+- Dont need to install
+- Just extract the folder and open the index.html
+
+
+
+## Authors
+
+👤 **Author1**
+cindy shin
+
+👤 **Author1**
```

Comment:
- [ ] OPTIONAL - Please, consider placing here your contact info, not the original design author.